docs/dev/guides/back-end-routes.md

```diff
@@ -88,31 +79,17 @@ Our route will render the template `my_backend_route.html.twig` which must be pl
 into `/templates`.
 
 ```twig
-{% extends "@ContaoCore/Backend/be_page.html.twig" %}
-
-{% block headline %}
-    Not only the content of the `title`-tag but also the title of the content section.
-{% endblock %}
+{% extends "@Contao/be_main" %}
 
-{% block error %}
-    Will be placed within the error block.
-{% endblock %}
+{% block main_headline %}Headline{% endblock %}
```

Comment:
There is no block `main_headline` - you need to set the `headline` via template parameter like `title`!